lib/.env

```diff
@@ -0,0 +1,2 @@
+SERVER_IP=10.66.172.59
+SERVER_PORT=8080
```

Comment:
The .env does not need to be part of the PR, right?

Reply:
fixed